src/commands/appdistribution-testers-add.ts

```diff
@@ -5,13 +5,24 @@ import { AppDistributionClient } from "../appdistribution/client";
 import { getEmails, getProjectName } from "../appdistribution/options-parser-util";
 
 export const command = new Command("appdistribution:testers:add [emails...]")
-  .description("add testers to project")
+  .description("add testers to project (and possibly group)")
   .option("--file <file>", "a path to a file containing a list of tester emails to be added")
+  .option(
+    "--group-alias <group-alias>",
+    "if specified, the testers are added to the group identified by this alias"
```

Comment:
nit: Clarify that this means they will _also_ be added to the group in addition to the project, given testers:remove has different behavior

Reply:
done